nix/deps/clojure/deps.json

```diff
@@ -11,7 +11,6 @@
       "sha256": "193i0xgn295bdz5gk4zrk4ncfa29x8vl97bfd01kl1za44ixm6wq"
     }
   },
-
```

Comment:
Please do not commit such pointless changes.

Reply:
Yes, sorry, VS Code must have automatically created this